packages/web/src/pages/settings-page/components/desktop/AuthorizedApps/AuthorizedAppListItem.tsx

```diff
@@ -0,0 +1,89 @@
+import { useCallback, useMemo } from 'react'
```

Comment:
Lots of custom styles here - again I think they might be copy pasted but would prefer not to regress in patterns otherwise

Reply:
yeah they're all copy pasted